libs/page/api/src/lib/page.service.ts

```diff
@@ -202,12 +202,15 @@ export class PageService {
       }
     })
 
+    const pagePublishedAt =
```

Comment:
Fixes https://wepublish.atlassian.net/browse/TSRI-134

When the page.publishedAt doesn't exist or is older than the new publishing date, take the new publishing date.